cypress/integration/commits/task_status_icons.ts

```diff
@@ -1,36 +1,30 @@
 describe("Waterfall Task Status Icons", () => {
   before(() => {
     cy.login();
+    const route =
+      "/commits/evergreen?chartType=percentage&statuses=all,failed-umbrella,failed,task-timed-out,test-timed-out,known-issue,success,running-umbrella,started,dispatched,scheduled-umbrella,will-run,pending,unstarted,system-failure-umbrella,system-failed,system-timed-out,system-unresponsive,setup-failed,inactive";
     cy.visit(route);
   });
 
-  it("Grouped status icons should link to the version page with appropriate table filters", () => {
-    const hrefs = [
-      "/version/5e4ff3abe3c3317e352062e4/tasks?statuses=success&variant=%5Elint%24",
-      "/version/5e4ff3abe3c3317e352062e4/tasks?statuses=running-umbrella,started,dispatched&variant=%5Elint%24",
-      "/version/5e4ff3abe3c3317e352062e4/tasks?statuses=success&variant=%5Eubuntu1604%24",
-      "/version/5e4ff3abe3c3317e352062e4/tasks?statuses=running-umbrella,started,dispatched&variant=%5Evariant%24",
-      "/version/evergreen_33016573166a36bd5f46b4111151899d5c4e95b1/tasks?statuses=success&variant=%5Eubuntu1604%24",
-    ];
-    cy.dataCy("grouped-task-status-badge").each(($el, index) => {
-      cy.wrap($el)
-        .find("a")
-        .should("have.attr", "href")
-        .and("equals", hrefs[index]);
-    });
+  beforeEach(() => {
+    cy.preserveCookies();
   });
 
-  it("Single task status icons should link to the corresponding task page", () => {
-    const hrefs = [
-      "/task/evergreen_ubuntu1604_test_annotations_b_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48",
-      "/task/evergreen_ubuntu1604_test_cloud_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48",
-      "/task/evergreen_ubuntu1604_test_service_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48",
-    ];
-    cy.dataCy("task-status-icon").each(($el, index) => {
-      cy.wrap($el).should("have.attr", "href").and("equals", hrefs[index]);
+  describe("Grouped Icons", () => {
+    it("Grouped status icons should be rendered on waterfall page", () => {
+      cy.dataCy("grouped-task-status-badge")
+        .first()
+        .scrollIntoView()
+        .should("be.visible");
     });
   });
 
-  const route =
-    "/commits/evergreen?chartType=percentage&statuses=all,failed-umbrella,failed,task-timed-out,test-timed-out,known-issue,success,running-umbrella,started,dispatched,scheduled-umbrella,will-run,pending,unstarted,system-failure-umbrella,system-failed,system-timed-out,system-unresponsive,setup-failed,inactive";
+  describe("Single task status icon", () => {
+    it("Single task status icons should be rendered on the waterfall page", () => {
+      cy.dataCy("task-status-icon")
+        .first()
+        .scrollIntoView()
+        .should("be.visible");
```

Comment:
I think this is okay since the goal of this test is to see that the task status icons exist on the waterfall page. This example is also presented in the [cypress docs](https://docs.cypress.io/api/commands/scrollintoview#Assertions) and it may be possible that the item is not visible after scrolling. I test href generation in the jest test since we thought it could be brittle to test that in cypress ([thread](https://github.com/evergreen-ci/spruce/pull/910#discussion_r724463638))